interface/themes/core/patient/demographics.scss

```diff
@@ -34,3 +35,116 @@
     }
   }
 }
+
+
+#DEM table {
+  width: 100%;
+}
+
+#DEM .label_custom {
+  text-align: right;
+}
+
+#DEM .data {
+  padding-left: 8px;
+  padding-right: 15px;
+}
+
+
+/* Demographics - Previous Addresses - address_list_display.php */
+
+#DEM .table_text_addresses {
+  border-bottom: 2px solid #e9ecef;
+  margin-top: 1em;
+}
+
+#DEM .text_addresses.header {
```

Comment:
I'd bootstrap style you addresses widget so you're not tied to the DEM id in case the widget is used elsewhere which the way it is shaping up, is very likely.

Reply:
@sjpadgett
Based on weekly conference call discussion, I addressed this.  I got rid of all of the styling that I put in demographics.css and used bootstrap classes & inline classes.